src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -160,6 +160,28 @@
   <data name="AvoidExpectedExceptionAttributeTitle" xml:space="preserve">
     <value>Avoid '[ExpectedException]'</value>
   </data>
+  <data name="DataRowShouldBeValidDescription" xml:space="preserve">
+    <value>DataRow attribute should have the following layout to be valid:
```

Comment:
Shall we say `DataRow entry` instead of `DataRow attribute`?

Reply:
Fixed.